code-studio/.istanbul.yml

```diff
@@ -1,10 +1,11 @@
 instrumentation:
   root: src/js
-  include-all-sources: true
+  # include-all-sources can't be true because mochify-istanbul defaults to false and doesn't respect this file.
+  include-all-sources: false
```

Comment:
Also, I wanted to explicitly set it to a value in case the default for check coverage changes.